provision-contest/ansible/admin.yml

```diff
@@ -151,3 +137,33 @@
         etype: user
         permissions: rwx
         state: present
+
+    - name: Download offline repos
+      when: WF_RESTRICTED_NETWORK
+      become: true
+      become_user: domjudge
+      block:
+        - name: Add ccsadmin remote
+          git_config:
+            name: "remote.{{ item.remote }}.url"
+            value: "{{ item.url }}"
+            scope: local
+            repo: /home/domjudge/domjudge-checkout
+          loop:
```

Comment:
Is this the right order?

Reply:
Order does not matter for yaml, I checked and this does create those remotes.

Reply:
Which one is the default remote?

Reply:
We first clone first so that would be the `origin`, the idea is to change the origin when another host comes available.

I verified on a VM and `origin` is the still default even after the other remotes get added.